supervisor/addons/addon.py

```diff
@@ -87,7 +87,6 @@ def __init__(self, coresys: CoreSys, slug: str):
         """Initialize data holder."""
         super().__init__(coresys, slug)
         self.instance: DockerAddon = DockerAddon(coresys, self)
-        self.state: AddonState = AddonState.UNKNOWN
```

Comment:
The move to model is wrong. In future I will split that more as just over the model. A store object have no state and they use also the model